File: src/main/java/net/ndrei/teslacorelib/test/CreativeGeneratorBlock.java
package net.ndrei.teslacorelib.test;

import net.ndrei.teslacorelib.TeslaCoreLib;
import net.ndrei.teslacorelib.blocks.OrientedBlock;

/**
 * Created by CF on 2016-12-27.
 */
public class CreativeGeneratorBlock extends OrientedBlock<CreativeGeneratorEntity> {
    public CreativeGeneratorBlock() {
        super(TeslaCoreLib.MODID, "creative_generator", CreativeGeneratorEntity.class);
    }
}

File: src/main/kotlin/net/ndrei/teslacorelib/test/CreativeGeneratorBlock.kt
package net.ndrei.teslacorelib.test

import net.ndrei.teslacorelib.TeslaCoreLib
import net.ndrei.teslacorelib.annotations.AutoRegisterBlock
import net.ndrei.teslacorelib.blocks.OrientedBlock
import net.ndrei.teslacorelib.config.TeslaCoreLibConfig

/**
 * Created by CF on 2017-06-27.
 */
@AutoRegisterBlock(TeslaCoreLibConfig.REGISTER_TEST_MACHINES)
object CreativeGeneratorBlock : OrientedBlock<CreativeGeneratorEntity>(TeslaCoreLib.MODID, TeslaCoreLib.creativeTab, "creative_generator", CreativeGeneratorEntity::class.java)
